package.json

```diff
@@ -28,26 +28,26 @@
   },
   "dependencies": {
     "@expo/samples": "2.1.1",
-    "expo": "^32.0.0",
-    "jest": "^24.8.0",
-    "node-libs-browser": "^2.2.0",
-    "prop-types": "^15.7.2",
+    "expo": "32.0.0",
+    "jest": "24.8.0",
+    "node-libs-browser": "2.2.0",
+    "prop-types": "15.7.2",
     "react": "16.5.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
-    "react-navigation": "^3.0.9",
-    "web3": "^1.0.0-beta.52"
+    "react-navigation": "3.0.9",
+    "web3": "1.0.0-beta.52"
```

Comment:
Try to use web3 in a stable version (1.2.1). You would have to check if there are no breaking changes.